Tabnine/src/main/java/com/tabnine/statusBar/TabnineIconProvider.kt

```diff
@@ -0,0 +1,29 @@
+package com.tabnine.statusBar
+
+import com.tabnineCommon.binary.requests.config.CloudConnectionHealthStatus
+import com.tabnineCommon.capabilities.CapabilitiesService
+import com.tabnineCommon.capabilities.Capability
+import com.tabnineCommon.general.ServiceLevel
+import com.tabnineCommon.general.StaticConfig
+import com.tabnineCommon.general.getSubscriptionType
+import javax.swing.Icon
+
+object TabnineIconProvider {
+    @JvmStatic
+    fun getIcon(
+        serviceLevel: ServiceLevel?,
+        isLoggedIn: Boolean?,
+        cloudConnectionHealthStatus: CloudConnectionHealthStatus
+    ): Icon {
+        if (!CapabilitiesService.getInstance().isReady || isLoggedIn == null || serviceLevel == null || (
+            CapabilitiesService.getInstance()
+                .isCapabilityEnabled(Capability.FORCE_REGISTRATION) &&
+                !isLoggedIn && serviceLevel == ServiceLevel.FREE
```

Comment:
if the user isn't logged in they are free, always, no?

Reply:
Yes, not logged in and Free meaning they should not have the starter icon, just Tabnine. (Same as vscode)